develop/reference/articles/01-intro/00-development-reference-intro.markdown

```diff
@@ -1,113 +1,61 @@
 # Development Reference [](id=development-reference)
 
 <p style="margin-left: 40px;">
-Here you'll find Liferay's reference documentation for Java, tag libraries,
+This page gives you access to reference documentation for Java, tag libraries,
 JavaScript and CSS, and deployment descriptors.
 </p>
 
 <p style="margin-left: 40px;">
 Liferay Portal 7 Community Edition's Portal Core reference documentation is
 available online  at <a href="https://docs.liferay.com/portal/7.0"
-target="_blank">https://docs.liferay.com/portal/<span
+target="_blank">https://docs.liferay.com/portal/7.0<span
```

Comment:
No, we don't need these generic URLs. I'm removing this paragraph and the next. Thanks.